targets/csharp-unityv2/Testing/Tests/Admin/AdminEventTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class AdminEventTests : MonoBehaviour {
+
```

Comment:
Tests should not be MonoBehaviour's.
Delete this placeholder, or fill it out into at least some kind of partial test.
Putting this here as a MonoBehaviour is misleading.